dash/pages/poc.py

```diff
@@ -0,0 +1,140 @@
+from datetime import date
+from pathlib import Path
+import dash
+import plotly.graph_objects as go
+from dash import html, dcc, callback, Output, State, Input, dash_table
+from pandas import DataFrame
+from ted_sws.data_manager.adapters.triple_store import FusekiAdapter
+
+dash.register_page(__name__)
+
+TED_DATA_MONGODB_DATABASE_NAME = 'ted_analytics'  # TODO: temporary while not put in config resolver
+QUERY_FOLDER = 'sparql_queries'
+TRIPLE_STORE_ENDPOINT_NAME = 'ted_data_dataset'
+SPARQL_QUERY_FORMAT = '.rq'
+FILES_ENCODING = 'utf-8'
+QUERY_TIME_FORMAT = 'yyyy-mm-dd'
+MIN_DATE = date(1995, 8, 5)
+MAX_DATE = date(2030, 9, 19)
+START_DATE = date(2020, 8, 5)
+END_DATE = date(2022, 8, 25)
+
+fuseki_adapter = FusekiAdapter()
+fuseki_endpoint = fuseki_adapter.get_sparql_triple_store_endpoint(TRIPLE_STORE_ENDPOINT_NAME)
+
+
+def figure_q14():
+    collection_name = 'q14'
+    query = Path(f'{QUERY_FOLDER}/{collection_name}{SPARQL_QUERY_FORMAT}').read_text(encoding=FILES_ENCODING)
+
+    query_result: DataFrame = fuseki_endpoint.with_query(query).fetch_tabular()
+
+    if query_result.empty:
+        return go.Figure(go.Indicator())
+
+    fig = go.Figure(go.Indicator(mode="number", value=query_result['Nr_of_SME'][0], title={"text": "Nr of SME"},
+                                 delta={'position': "top", 'reference': 320}, domain={'x': [0, 1], 'y': [0, 1]}))
+    return fig
+
+
+@callback(
+    Output('q15_graph', 'figure'),
+    State('q15_graph', 'figure'),
+    State('q15_daterange', 'start_date'),
+    State('q15_daterange', 'end_date'),
+    Input('q15_button', 'n_clicks'),
+    prevent_initial_call=True,
+)
+def figure_q15(figure: dict = None, start_date: str = START_DATE.strftime(QUERY_TIME_FORMAT),
```

Comment:
in the MVP we want a better way of organising these aspects.

Reply:
Perhaps are fine for now in the POC.

Reply:
Yes. This is for POC, to have a pipeline from implementation to result on our page. Further, it remains to be arranged correctly